apiserver/common/tools.go

```diff
@@ -297,7 +297,7 @@ func (f *ToolsFinder) matchingStorageTools(args params.FindToolsParams) (coretoo
 	for i, m := range allMetadata {
 		vers, err := version.ParseBinary(m.Version)
 		if err != nil {
-			return nil, errors.Annotatef(err, "unexpectedly bad version %q in tools storage", m.Version)
+			return nil, errors.Annotatef(err, "unexpectedly bad version %q in agent binaries storage", m.Version)
```

Comment:
unexpected bad version %q in storage of agent binaries?
I like
unexpected bad version
at the least

Reply:
'unexpected bad version %q of agent binary in storage'....?

Reply:
sgtm